### PR TITLE
Submission: Add evaluation result of RAFT-30B-A3B on retail domain

### PR DESCRIPTION
**Pull request by Ziyiii0-0**
### Overview
- **Model**: RAFT-30B-A3B
- **Base model**: Qwen3-30B-A3B-Thinking-2507, fine-tuned on synthetic retail-domain trajectories
- **Submission date**: 2026-04-29
- **Domain**: Retail
### Results — Retail Domain
| Metric   | Score   |
|----------|---------|
| pass@1   | 82.46%  |
| pass@2   | 72.95%  |
| pass@3   | 66.23%  |
| pass@4   | 61.40%  |
### Link to Trajectory
https://huggingface.co/datasets/Ziyiii0-0/Trajectory2Task-30b-a3b-taubench/blob/main/results.json
### Evaluation Setup
- **Trials**: 4
- **User simulator**: `claude-sonnet-4-5-20250929`
- **Agent LLM**: `Qwen3-30B-A3B-Thinking-2507` fine-tuned checkpoint (hosted via vLLM)
### Methodology Notes
Training data was collected on a fully synthetic retail database whose entities (users, products, orders) are **disjoint** from the τ-bench evaluation database.
- Modified prompts: No
- Omitted questions: No

### CODE REVIEW

**Comment by benshi34** [comment]
Thanks for the submission! Generally this looks good to add -- wanted to check if there was an associated paper or blog post to link to where the methodology is documented in more detail?

**Comment by Ziyiii0-0** [comment]
Hi, thanks for the reply! We'll post the blog in one or two days and add the link here.

**Comment by Ziyiii0-0** [comment]
Hi again, thanks for taking time to review. We've add the blog post link into the submission file, also [here](https://raft.hailab.io/), and updated our submission name.

**Comment by benshi34** [comment]
Great, I'll merge this today. Great work!
